index.html

```diff
@@ -573,12 +573,61 @@ <h1>{{ { US: 'USA', Intl: 'International' }[chosen_domain] }}
   "#a7f432", "#a83731", "#af6e4d", "#ff007c", "#fcd667", "#bfafb2", "#c46210",
   "#4a646c", "#a6e7ff", "#ffd0b9", "#45a27d", "#3aa8c1"];
 
+// List of style elements we have created
+var styleNodes = [];
+
+// The current mapping from locality name to color index
+var currentMapping = {}
```

Comment:
`currentMapping` isn't descriptive enough a name after we've forgotten about this problem.  Maybe `currentLocalityToColorIndex`.

Reply:
Done.